MAINTAINERS.md

```diff
@@ -0,0 +1,39 @@
+## Core Maintainers of this repository
+
+| Name                  | Email                       | Kubernetes Slack         | GitHub                                            | Company           |
+|-----------------------|-----------------------------|--------------------------|---------------------------------------------------|-------------------|
+| Frederic Branczyk     | fbranczyk@gmail.com         | `@brancz`                | [@brancz](https://github.com/brancz)              | Red Hat           |
+| Lili Cosic            | coscilili@gmail.com         | `@lili`                  | [@lilic](https://github.com/lilic)                | Red Hat           |
+| Matthias Loibl        | mail@matthiasloibl.com      | `@metalmatze`            | [@metalmatze](https://github.com/metalmatze)      | Red Hat           |
+| Paweł Krupa           | paulfantom@gmail.com        | ``                       | [@paulfantom](https://github.com/paulfantom)      | Red Hat           |
```

Comment:
@paulfantom can you tell me your slack k8s name, thanks!

Reply:
It's `paulfantom` (everywhere :D )